src/scripts/components/VideoOverlay.js

```diff
@@ -162,6 +158,7 @@ const CentralizedContent = styled.div`
 `
 
 const StartScreenIcon = styled.span`
+  color: ${props => props.theme.colors.button.warn};
```

Comment:
Can this be a `<Span warn />`?

Reply:
Besides color it doesn't share any other style. And perhaps it's should change.
I've change the color to `props.theme.colors.text.accent` https://github.com/Paratii-Video/paratii-portal/pull/769/commits/63bb45bcaa6563763fc35d97f0a4e543dea67a4f